pkg/apis/enterprise/v3/common_types.go

```diff
@@ -276,6 +276,18 @@ type AppFrameworkSpec struct {
 	//    3. If anything more than the max value is specified then we set it to 1 day.
 	AppsRepoPollInterval int64 `json:"appsRepoPollIntervalSeconds,omitempty"`
 
+	// Period after which the scheduler tries to yield from the current reconcile entry
+	// +kubebuilder:validation:Optional
+	// +kubebuilder:default:=90
+	// +kubebuilder:validation:Minimum:=30
+	SchedulerYieldInterval uint64 `json:"schedulerYieldIntervalSeconds,omitempty"`
+
+	// Max. number of tries used by the scheduler for a pipeline phase
+	// +kubebuilder:validation:Optional
+	// +kubebuilder:default:=3
+	// +kubebuilder:validation:Minimum:=1
+	PhaseMaxTries uint32 `json:"phaseMaxTries,omitempty"`
```

Comment:
Are we doing any validation for these two params in lets say `ValidateAppFrameworkSpec` ?

Reply:
Kubebuilder will validate it for us, so, we are not doing any explicit validation.